extensions/amp-sidebar/1.0/component.js

```diff
@@ -135,42 +136,93 @@ function SidebarWithRef(
   }, [opened, close]);
 
   return (
-    mounted && (
-      <>
-        <ContainWrapper
-          as={Comp}
-          ref={sidebarRef}
-          size={false}
-          layout={true}
-          paint={true}
-          part="sidebar"
-          wrapperClassName={`${classes.sidebar} ${
-            classes.defaultSidebarStyles
-          } ${side === Side.LEFT ? classes.left : classes.right}`}
-          role="menu"
-          tabindex="-1"
-          hidden={!side}
-          {...rest}
-        >
-          {children}
-        </ContainWrapper>
-        <div
-          ref={backdropRef}
-          onClick={() => close()}
-          part="backdrop"
-          style={backdropStyle}
-          className={`${backdropClassName ?? ''} ${classes.backdrop} ${
-            classes.defaultBackdropStyles
-          }`}
-          hidden={!side}
-        >
-          <div className={classes.backdropOverscrollBlocker}></div>
-        </div>
-      </>
-    )
+    <div className={objstr({[classes.unmounted]: !mounted})}>
+      <ContainWrapper
+        as={Comp}
+        ref={sidebarRef}
+        size={false}
+        layout={true}
+        paint={true}
+        part="sidebar"
+        wrapperClassName={objstr({
+          [classes.sidebar]: true,
+          [classes.defaultSidebarStyles]: true,
+          [classes.left]: side !== Side.RIGHT,
```

Comment:
nit: Since there isn't a side value other than `LEFT` and `RIGHT`, let's make this more readable with:
```suggestion
          [classes.left]: side === Side.LEFT,
```

Reply:
Thanks for the suggestion.  The one problem with this is if the user passes in `side` that is not equal to `Side.RIGHT` or `Side.LEFT` then neither class will be included.  Weird corner case I know.